servicetalk-grpc-protobuf/src/main/java/io/servicetalk/grpc/protobuf/ProtoBufSerializationProviderBuilder.java

```diff
@@ -172,7 +172,7 @@ public GrpcSerializationProvider build() {
             @SuppressWarnings("unchecked")
             HttpSerializer<T> httpSerializer = serializersForType.get(codec);
             if (httpSerializer == null) {
-                throw new MessageEncodingException("Unknown encoding: " + codec.name());
```

Comment:
Perhaps instead of the encoding name, which may be non-canonical, the exception could be created with the `ContentCodec` instance responsible.

Reply:
This PR mostly targets 0.41 branch. In the main branch, old serialization is already deprecated. This exception type was missed, but will be deprecated by #1924. Will minimize changes for the code that is going away soon.